conda/conda_build_config.yaml

```diff
@@ -1,5 +1,5 @@
 python:
-  - 3.8
+  - 3.10
 
 setuptools:
-  - 62.0
+  - 65.0
```

Comment:
Out of interest, do you know what problem is tying us to setuptools v65 ?

Reply:
There used to be a problem with newer setuptools versions but it can be unpinned now.